docs/wallet/Desktop-Wallet-Guide.md

```diff
@@ -40,3 +40,68 @@ Due to the nature of UTXO, every time a transaction is made a couple of new Unsp
+
+The Passphrase is an additional optional 25th word that you are free to choose for yourself. It can be composed of any lower/upper-case character, number, and/or signs and be as long as you want.
+
+Once you create a wallet into the desktop wallet app, the next time you open the app to log in to your wallet, you will be able to provide an additional Passphrase. **It is important to remember that any unique Passphrase will generate and give access to a completely new wallet.**
+
+### Example scenario
+
+#### The "normal wallet"
+
+Let's assume that you created a new wallet, you gave it the name _My Alephium Wallet_ and the password _myVeryStrongPassw0rd_. Let's also assume that the generated 24-word secret phrase is `scan pause slender around cube flavor neck shrug gadget ramp rude lend capable tone nose unhappy gift across cluster minor tragic fever detail script`. You decide to transfer 100 ALPH to it. We'll call this the "normal wallet".
+
+#### The "hidden wallet"
+
+The next time you open your wallet, you select that you want to login to _My Alephium Wallet_ and you provide your password _myVeryStrongPassw0rd_. Instead of clicking _Login_ directly, however, you enter the Passphrase _letsUnlockAHiddenWallet_, and then click _Login_. This gives you access to a completely new wallet (let's call it "hidden wallet"). You will not be able to see your balance of 100 ALPH there, since the wallet you tranferred your 100 ALPH to was the one that did not use the Passphrase _letsUnlockAHiddenWallet_ (a.k.a. the "normal wallet"). You decide to transfer 10'000 ALPH to the "hidden wallet".
+
+#### The other "hidden wallets"
+
+When you logout and you try to login into your "hidden wallet" again, if you enter your chosen Passphrase wrong (for example you enter _letsUnlockAHiddenwallet_ instead of _letsUnlockAHiddenWallet_ - notice the lower case `w` instead of `W`), you will gain access to yet another new "hidden wallet" with 0 funds. Logging out and logging in again, with the correct Passphrase this time, will give you access to your "hidden wallet" with the 10'000 ALPH in it.
+
+### Plausible deniability use-case
+
+In the unfortunate scenario where an attacker forces you to reveal your wallet password and your 24-word secret phrase (also known as the "$5 wrench attack"), the Passphrase feature can help you secure your physical safety as well as most of your wealth by convincing the attacker that they have access to all your funds. Let's explore this with the use of an example:
+
+- You are a proud owner of 10'000 ALPH.
+- You have created a "normal wallet" (no Passphrase) with the name _My Alephium Wallet_ and the password _myVeryStrongPassw0rd_. You have transferred 100 ALPH to it.
```

Comment:
Yes, may make sense to explain that the password is used only on this computer to encrypt the storage of the 24 words (mnemonic phrase) while the passphrase add a word to the 24, thus producing a completely different wallet.

it is safe to store the password near the 24 words since the password is only needed on the computer to decrypt the storage of those 24 words, if you find them on a paper, you do not need the password anymore. 

It is also a good idea to mention that the passphrase should NOT be stored near the 24 words, but should also absolutely be backedup. (in a different location) and also not on the computer / phone hosting the wallet.

Reply:
Thanks, guys, I've updated the "Important Takeaways"